packages/create/src/utils.ts

```diff
@@ -153,3 +153,35 @@ stdout: ${stdout}`,
     });
   });
 }
+
+export class ProjectManager {
+  public packageManager: PackageManager;
+
+  constructor() {
+    this.packageManager = getPackageManager();
+  }
+
+  private getPackageRunner(): string {
```

Comment:
Small nit: since this value isn't actually dynamic, we can move this to the constructor:

```ts
const PM_TO_RUNNER_MAP = {
  "yarn": "yarn dlx",
  // etc.
} as const;

// in constructor: 
this.runner = PM_TO_RUNNER_MAP[this.packageManager];
```

Or something like that. Then just `${this.runner}` in your string builder.